src/pages/components/CartWishCard/CartWishCard.jsx

```diff
@@ -143,21 +153,43 @@ function moveToWishlistOrCart(
     setIsLoading,
     cart,
     wishlist,
-    _id
+    _id,
+    setToastMessage
 ) {
     deleteFromCartOrWishlist(
         cardType === "cart" ? "cart" : "wishlist",
```

Comment:
instead of using strings, create a constants file and add the strings to it, this will prevent you from making any spelling mistakes